faqs/galaxy/re_use_equivalent_jobs.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How do I re-use equivalent jobs in Galaxy (aka Job Cache)?
+area: user preferences
+layout: faq
+box_type: tip
+contributors: [dadrasarmin]
+---
+
+Galaxy instances allow you to retrieve the results of running a tool from the cache (and hence almost immediately) if the tool with the exact same input and parameters has already been executed in your history. 
+This is also true if another user used the same tool and inputs and made their history publicly available.
+This is highly helpful in situations like training events. In such a case, the instructor might complete the tasks and make their results public. 
```

Comment:
```suggestion
This can be highly helpful, e.g., for training events, if the instructor makes a respective training history public before the event. 
```